docs/guides/postgres/custom-versions/setup.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Setup Custom PostgresVersions
+menu:
+  docs_0.8.0:
+    identifier: pg-custom-versions-setup-postgres
+    name: HA Setup
+    parent: pg-custom-versions-postgres
+    weight: 10
+menu_name: docs_0.8.0
+section_menu_id: guides
+---
+> New to KubeDB? Please start [here](/docs/concepts/README.md).
+
+## Setting up Custom PostgresVersions
+
+PostgresVersions are KubeDB crds that define the docker images KubeDB will use when deploying a postgres database.
+
+## Creating a Custom Postgres Database Image for KubeDB
+
+The best way to create a custom image is to build ontop of the existing kubedb image
+
+```docker
+FROM kubedb/postgres:10.2
```

Comment:
On a unreleased note to this pr, should we create a `github.com/kubedb/timescaledb` repo and just include the docker files and add then to the catalog ?